### PR TITLE
Feature/dfs 129 hotfix v1

### PR DESCRIPTION
**Pull request by AleksDoesCode**
Added minor improvements and bugfixes

### CODE REVIEW

File: lib/pages/dashboard/portfolio_balance_card.dart [hunk 1]
```diff
@@ -67,14 +67,15 @@ class PortfolioBalanceCard extends HookWidget {
...
                           children: [
                             Text(
-                              AppLocalizations.of(context)!
-                                  .dash_portofolio_change,
-                              style: Theme.of(context).textTheme.labelSmall,
-                            ),
+                                AppLocalizations.of(context)!
+                                    .dash_portofolio_change,
+                                style: Theme.of(context).textTheme.labelSmall),
                             SizedBox(
```
Comment (TilDoesCode) [inline lib/pages/dashboard/portfolio_balance_card.dart:77]: Can be removed

File: lib/widgets/cards/recently_closed_order_card.dart [hunk 2]
```diff
@@ -76,9 +76,7 @@ class RecentlyClosedOrderCard extends HookWidget {
                             crossAxisAlignment: CrossAxisAlignment.end,
                             children: [
                               Text(
-                                data.difference > 0
-                                    ? data.tokenAmmount.toStringAsFixed(2)
-                                    : "- ${data.tokenAmmount.toStringAsFixed(2)}",
+                                data.difference.toStringAsFixed(2),
```
Comment (TilDoesCode) [inline lib/widgets/cards/recently_closed_order_card.dart:79]: Use proper rounding

**Review by TilDoesCode — CHANGES_REQUESTED** [review]

File: lib/pages/dashboard/portfolio_balance_card.dart (hunk 1 above)
Comment (TilDoesCode) [reply lib/pages/dashboard/portfolio_balance_card.dart:77]: Refers to the SizedBox starting in l77

**Comment by AleksDoesCode** [comment]
Fixed

**Review by TilDoesCode — APPROVED** [review]

**Comment by AleksDoesCode** [comment]
Added Version counter to account

**Pull request merged**